### ISSUE TITLE
How to import all the mails and folders?

### ISSUE THREAD

**Issue by gauthierbuttez** [thundernest/import-export-tools-ng#90]
Hi,

I made a backup of all my mails and folders:
http://prntscr.com/rhd7qi

I tried to import by making a right click on the inboix folder>ImportExport NG> 
      >Import mbox doesn't work
      >Import messages doesn't work
      >Import all messages from a directory doesn't work

I have thunderbird 68.6.0 .

Can you help me please?

**Comment by cleidigh** [comment]
@gauthierbuttez 
Thanks for the post sorry for the issue...

What method did you use for the Backup?
If you exported to mbox then you can use the import mbox with structure option.
There has always been a gap in the fact that there is no user guide, but I am slowly trying to fix that.
Also there have been issues with large imports I am also trying to deal with.
@cleidigh

**Comment by JardaCZE** [comment]
I have same issue. 
I exported emails (Export all the folders with structure) and I am unable to import it back (tested every option I have found). 

Tool version 4.0.4
Thunderbird 68.7.0 (32 bit)

**Comment by JardaCZE** [comment]
I think I have found what is the problem. 

You cannot import directly into the IMAP folder. 
I have to import to the TEMP folder and move the messages. 

https://support.mozilla.org/en-US/questions/1164751

**Comment by cleidigh** [comment]
@JardaCZE 
Yes, since the IMAP folders are technically "remote" they are not meant to be used for importing.
There should be a pop-up alert for this.  Did that not appear?

**Comment by JardaCZE** [comment]
No POP-UP alert.
It starts importing. In the left bottom corner it was the message "Start
importing" and it has nearly frozen my Thunderbird.

But something must happen.
Local file size on my C: drive has grown, but no message has appeared in
Thunderbird.


Příjemný den Vám přeje,

Jaroslav Mixa, DiS
mobil: (+420) 773 664 554 ; e-mail: jaroslav.mixa@czisp.cz ; www:
www.czisp.cz
--------------------------------------------------------------------------------------------
Žádáme Vás, abyste ve Vaší odpovědi ponechali původní text mailu.
--------------------------------------------------------------------------------------------


st 29. 4. 2020 v 21:54 odesílatel Christopher Leidigh <
notifications@github.com> napsal:

**Comment by cleidigh** [comment]
@gauthierbuttez 
@JardaCZE 
I have been working to catch several tricky issues with EML for the import.  if you would like to try the test extension
download and install locally.  the third option under the folder menu (experimental) has all my current fixes
https://github.com/cleidigh/iet-tests/blob/master/xpi/iet-ng-tests-0.1.14-tb.xpi
feedback welcome!
@cleidigh

**Comment by JardaCZE** [comment]
Have no chance to test it now.


>
